executable/pom.xml

```diff
@@ -68,9 +68,12 @@
 								<transformer
 										implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
 								<transformer
-										implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
-									<mainClass>com.bakdata.conquery.ConqueryServer</mainClass>
-								</transformer>
+                                        implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                    <mainClass>com.bakdata.conquery.ConqueryServer</mainClass>
+                                    <manifestEntries>
+                                        <Multi-Release>true</Multi-Release>
```

Comment:
was genau macht das hier

Reply:
Der HANA JDBC Driver kommt als Multi-Release-Jar. Die benötigte Version wird wenn dieses Setting deaktiviert ist, nicht mit in die Uber-JAR aufgenommen.